### PR TITLE
revert(lib/runtime): rollback wasmer update `PR#3168`

### PR DESCRIPTION
**Pull request by EclesioMeloJunior**
## Changes

- Rollback changes made at https://github.com/ChainSafe/gossamer/pull/3168
- The changes update Gossamer to use `wasmerio/wasmer-go`, but we notice that Gossamer was running out of memory fell blocks synced
- After some profiling and investigation we could find that the `wasmer-go` was causing a weird memory leak and we opened an issue in its repository 
  - https://github.com/wasmerio/wasmer-go/issues/393

## Tests

With the `wasmer-go` updates the instance memory usage was reaching 100% fell blocks synced:
image

Currently, after the rollback we've synced more than 1 million blocks and the memory usage stills ok!

image

## Primary Reviewer

@timwu20

### CODE REVIEW

**Review by dimartiro — APPROVED** [review]

**Review by timwu20 — APPROVED** [review]

**Comment by timwu20** [comment]
Thanks for doing this @EclesioMeloJunior!

File: lib/runtime/wasmer/exports_test.go [hunk 1]
```diff
@@ -956,6 +977,10 @@ func TestInstance_PaymentQueryInfo(t *testing.T) {
 			continue
 		}
 		require.NoError(t, err)
+
+		fmt.Println(info.PartialFee.String())
```
Comment (jimjbrettj) [inline lib/runtime/wasmer/exports_test.go:981]: remove prints?

File: lib/runtime/wasmer/helpers.go [hunk 2]
```diff
@@ -3,24 +3,19 @@
 
 package wasmer
 
+// #include <stdlib.h>
+import "C" //skipcq: SCC-compile
```
Comment (jimjbrettj) [inline lib/runtime/wasmer/helpers.go:7]: :(

**Review by jimjbrettj — APPROVED** [review]
Sad but necessary, thank you for doing this

**Pull request merged**